build/EC.proj

```diff
@@ -59,63 +59,63 @@
 		<MSBuild Projects="$(projroot)/src/ECInterfaces/ECInterfaces-mono.csproj"
 				 Targets="$(msbuild-target)"
 				 Properties="$(msbuild-props)"
-				 ToolsVersion="3.5"/>
+				 ToolsVersion="15.0"/>
```

Comment:
We should omit `ToolsVersion`. Specifying it causes problems with Mono 6, and according to Microsoft developers it's not needed and doesn't do what you think it does. So best to leave it out.

Reply:
Done